CHANGELOG.md

```diff
@@ -11,7 +11,9 @@ All notable changes to this project will be documented in this file.
 - `frequency_average` method on UVData to average data along the frequency axis.
 
 ### Fixed
+<<<<<<< HEAD
```

Comment:
looks like this got left in after a rebase

Reply:
Also, the changes in this PR should be listed at the top (reverse chronological order). FYI there's another PR about to be merged that is likely to conflict with this (#790), so you might hold off on rebasing until that one is merged.

Reply:
Ok!

Reply:
that one has been merged, so I think you're good to go on a fresh rebase.

Reply:
Done